### PR TITLE
[#177] Fix: spotbug issue TestFunctionMetadata

### PR DESCRIPTION
**Pull request by SharrDesai**
### What type of PR is this?
Fix spot bug issue.

Uncomment only one /kind <> line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug /kind task /kind feature

### Which issue(s) this PR fixes:

Fixes #177 

### Special notes for your reviewers:

### CODE REVIEW

**Comment by RajeevRastogi** [comment]
lgtm

File: hetu-hive-functions/src/test/java/io/hetu/core/hive/dynamicfunctions/TestFunctionMetadata.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public void testGetFunctionNameWithException()
     {
         FunctionMetadata funcMetadata = new FunctionMetadata(
                 "isemptyio.hetu.core.hive.dynamicfunctions.examples.udf.IsEmptyUDF");
-        funcMetadata.getFunctionName();
+        assertEquals(IsEmptyUDF.class, funcMetadata.getClazz());
```
Comment (Nitin-Kashyap) [inline hetu-hive-functions/src/test/java/io/hetu/core/hive/dynamicfunctions/TestFunctionMetadata.java:62]: fix is not appropriate.. should be something like
```Java
 assertEquals(funcMetadata.getFunction(), "IsEmptyUDF")
```

File: hetu-hive-functions/src/test/java/io/hetu/core/hive/dynamicfunctions/TestFunctionMetadata.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ public void testGetClassNameWithException()
     {
         FunctionMetadata funcMetadata = new FunctionMetadata(
                 "isemptyio.hetu.core.hive.dynamicfunctions.examples.udf.IsEmptyUDF");
-        funcMetadata.getClassName();
+        assertEquals(IsEmptyUDF.class, funcMetadata.getClazz());
```
Comment (Nitin-Kashyap) [inline hetu-hive-functions/src/test/java/io/hetu/core/hive/dynamicfunctions/TestFunctionMetadata.java:88]: fix is not appropriate.. should be something like
```java
 assertEquals(funcMetadata.getFunction(), "IsEmptyUDF")
```

**Pull request merged**